docs/framework/guides/architecture/editing-engine.md

```diff
@@ -159,9 +159,11 @@ Markers are a special type of ranges.
 * They can only be created and changed through the {@link module:engine/model/writer~Writer model writer}.
 * They can be synchronized over the network with other collaborating clients.
 * They are automatically updated when the document's structure is changed.
-* They can be converted to attributes or elements in the [view](#view).
+* They can be {@link module:engine/conversion/downcasthelpers~DowncastHelpers#markerToHighlight converted to the editing view}, to show them in the editor.
```

Comment:
What about `markerToElement()`? Shouldn't we mention both things:

```
	* They can be converted to the editing view, to show them in the editor (as {@link module:engine/conversion/downcasthelpers~DowncastHelpers#markerToHighlight highlights} or {@link module:engine/conversion/downcasthelpers~DowncastHelpers#markerToElement elements).
```

Reply:
For a moment that was deprecated so I didn't include it. But since it will not be deprecated after all, we may mention it.